logger/plainFormatter.go

```diff
@@ -14,13 +14,30 @@ func (pf *PlainFormatter) Output(line LogLineHandler) []byte {
 	}
 
 	level := LogLevel(line.GetLogLevel())
+	timestamp := displayTime(line.GetTimestamp())
+	loggerName := formatLoggerName(line.GetLoggerName())
+	message := formatMessage(line.GetMessage())
+	args := formatArgsNoAnsi(line.GetArgs()...)
+
+	const formatStringWithCorrelationElements = "%s[%s] %s %s %s %s\n"
```

Comment:
why not taking out these consts on the package level?

Reply:
Fixed - moved to `constants.go`. Of course, now both `consoleFormatter` and `plainFormatter` have leaked their privates to the whole package :)

Reply:
agree, but we do not have to scan later the code file in search of some consts